### DISCUSSION TITLE
Wiki pages not accessible

### DISCUSSION THREAD

**Discussion by Mozila4060** [fingerprintjs/fingerprintjs#675] [General]
Hi,

We are looking to optimize configuration & settings to get the most stable fingerprint possible. 
On some pages, I saw references to the Wiki pages with the relevant information. But pages are not accessible. 
For instance:

Note 2: Browser independent components Some components change when you switch to a different browser on the same device. See: https://github.com/fingerprintjs/fingerprintjs/wiki/Browser-independent-components

Note 3: Unstable options Some options change every time you refresh the page. See: https://github.com/fingerprintjs/fingerprintjs/wiki/Stable-components

Did Wiki move? Can somebody help me locate those pages?

**Comment by Finesse** [comment]
Hi! The wiki you are talking about is the wiki for version 2 of the library. It has been moved to https://github.com/fingerprintjs/fingerprintjs/tree/v2/wiki. The current library version is 3, and the wiki doesn't work for the current library version.

We'll review your stability question later.

**Reply by Mozila4060** [reply]
Thank you.

**Comment by makma** [comment]
Hi @Mozila4060 , thank you for your question, and welcome to the FingerprintJS community! As @Finesse has already mentioned, the current version of FingerprintJS is v3. From your referring to wiki v2, I assume you are using FPJS v2 - am I right? Please note, the currently supported version is v3, nevertheless, we can rarely fix serious bugs in v2.

I've tried some of the mentioned components and options, and I can confirm, the current state (for v3) is not in sync with the mentioned wiki. We can consider creating new documentation content for the v3 about stability - is it important for you? Would you find it valuable?

Needless to say, with OSS FingerprintJS, you have to make some compromises probably and choose between stability and sensitivity - this depends on your use case. Please take a look at our docs in the [Extending components section](https://github.com/fingerprintjs/fingerprintjs/blob/master/docs/extending.md#extending-fingerprintjs).  I believe this might help you with adjusting/customizing behavior for your specific scenario. Moreover, you can find additional info about [screen frame](https://github.com/fingerprintjs/fingerprintjs/issues/568) and [canvas stability](https://github.com/fingerprintjs/fingerprintjs/blob/master/docs/extending.md#canvas-stabilization).

I'd like to ask for providing more pieces of information about the specific components/options.
- What browser-independent components do change when you switch to a different browser on the same device?
- What specific options do change every time you refresh the page?

These information will help us to set proper expectations and help people with similar questions and issues.
Thank you!

**Reply by Mozila4060** [reply]
@makma Thank you for the detailed message.
We are using the v3. I referred to the V2 pages as this the only pages I found that are dealing with stability and best practices. 
If you would have the same for v3 it would be great!

Thanks for the links. We will examine them.

**Reply by makma** [reply]
Hello @Mozila4060! I've just filed the[ issue for collecting info about stable and independent components](https://github.com/fingerprintjs/fingerprintjs/issues/677). Feel free to add any related info and your observations about stability. Once we have enough data we will distill it and create documentation pages.

**Reply by Mozila4060** [reply]
Hi @makma, Thank you for the update. Our engineers and analysts are analyzing code and collected data, we will share the insights.
